Plan/src/main/java/com/djrapitops/plan/system/cache/VelocityCacheSystem.java

```diff
@@ -0,0 +1,21 @@
+package com.djrapitops.plan.system.cache;
+
+import com.djrapitops.plan.system.PlanSystem;
+
+/**
+ * CacheSystem for Velocity.
+ *
+ * Based on BungeeCacheSystem
+ *
+ * <p>
+ * Used for overriding {@link DataCache} with {@link VelocityDataCache}
+ *
+ * @author MicleBrick
+ */
+public class VelocityCacheSystem extends CacheSystem {
```

Comment:
Duplication of this class is unnecessary, as only benefit is having it in velocity specific class.
This can be sorted later by refactoring common classes into network specific names